lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -69,7 +69,11 @@ Function Install-PrereqModule {
             }
             else {
                 try {
-                    Install-Module -Name $Name -MinimumVersion $PrereqModules[$Name] -Force -WhatIf:$CheckMode | Out-Null
+                    if($host.Version -ge "5.1") {
```

Comment:
Instead of checking the PS version you should be checking if `Install-Module` has the `SkipPublisherCheck` parameter as even over PowerShell versions can have a newer PowerShellGet module installed. This could be done like;

```
$install_params = @{
    Name = $Name
    MinimumVersion = $PrereqModules[$Name]
    Force = $true
    WhatIf = $CheckMode
}
if ((Get-Command -Name Install-Module).Parameters.ContainsKey('SkipPublisherCheck')) {
    $install_params.SkipPublisherCheck = $true
}

Install-Module @install_params > $null
```